Unity-Technologies/com.unity.multiplayer.samples.bitesize
# Multiplayer Use cases for Unity 6 - usability improvements

### Description
- Removed runtime network stats monitor from the only scene that had it
- All inputs in various scenes are handled through the new Unity Input System
- The last page of the "Data/Event Synchronization tutorial" redirects to the "Proximity Checks" tutorial
- Multiplayer Play Mode window is now interactable on the last page of all tutorials

### How to test/validate

1. Clone this branch
2. Open it with unity 6000.0.17f1
3. Go through all tutorials, opening the Multiplayer Play Mode Window on the last page: check that it is always interactable
4. Check that there are no calls to the old input system methods (`Input.<method>`) in code
5. Check that no scene has Runtime Network Stats Monitor

### Issue Number(s)
- https://jira.unity3d.com/browse/MTT-9078
- https://jira.unity3d.com/browse/MTT-8682
- https://jira.unity3d.com/browse/MTT-8385

### Contribution checklist
